include/database.py

```diff
@@ -1,18 +1,57 @@
 # database.py
 #
 # This module provides a wrapper for basic database functionality.
+import sys
 import psycopg2
 
+
 def select(connectionString, sql, parameters):
+
     # Open the connection
-    connection = psycopg2.connect(connectionString)
+    try:
+        # connect_timeout for connectionString is 60 in the yml file
+        connection = psycopg2.connect(connectionString, connect_timeout = 5)
+
+    # operational error is raised when parameters passed to connect() are incorrect
+    # or if the server runs out of memory
+
+    except psycopg2.OperationalError as err:
+
+        #print(f' Error Connecting : {err}')
+        sys.stderr.write('Error Connecting: Connection Timed out')
+        # set connection to 'None' in case of error
+        #connection = None
+        sys.exit(1)
```

Comment:
Library code shouldn't cause a program to exit, recall that the directions in the issue are for the scripts to gracefully handle the exception raised. The flow should be to:
1. Catch the error from `psycopg2`
2. Write the error out to `stderr`
3. Raise an exception (e.g., `DatabaseError`) to be handled by the function caller